### ISSUE TITLE
Poor results

### ISSUE THREAD

**Issue by JakubReha** [NVIDIA/tacotron2#363]
Hi, I am training a new speaker with the pre-trained model, I have 3 hours of data but I had to decrease the batch size to 2 because of computational issues. The validation loss and training loss are at the same value 0.4 after 12 000 iterations, however the resulting audio is talking nonsense even after 30 000. I used google API for transcription. Is batch size the problem? Also the mel looks weird.
<img width="830" alt="Screenshot 2020-06-02 at 10 00 08" src="https://user-images.githubusercontent.com/44469049/83495610-334f6380-a4b8-11ea-9db2-93ff880e336c.png">

**Comment by aishweta** [comment]
You need more data at least 6 hours of data from transfer learning.

**Comment by ksaidin** [comment]
3 hrs of dataset is not that bad, it will eventually overfit tho.

I suggest to train on google colab so that you can have a higher batch size.
